data/mods/Magiclysm/recipes/recipe_potions.json

```diff
@@ -0,0 +1,146 @@
+    "book_learn": [ [ "alchemy_basic", 3 ] ],
+    "qualities": [ { "id": "CHEM", "level": 1 }, { "id": "CONCENTRATE", "level": 1 }, { "id": "MANA_INFUSE", "level": 1 } ],
+    "tools": [ [ [ "surface_heat", 10, "LIST" ] ] ],
+    "components": [
+      [
+        [ "gin", 7 ],
+        [ "rum", 7 ],
+        [ "single_malt_whiskey", 7 ],
+        [ "tequila", 7 ],
+        [ "triple_sec", 7 ],
+        [ "vodka", 7 ],
+        [ "whiskey", 7 ],
+        [ "brandy", 7 ],
+        [ "mixed_alcohol_strong", 7 ]
+      ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "result": "cats_grace_potion",
+    "category": "CC_ENCHANTED",
+    "subcategory": "CSC_ENCHANTED_POTIONS",
+    "skill_used": "cooking",
+    "skills_required": [ "spellcraft", 5 ],
+    "difficulty": 5,
+    "time": "60 m",
+    "batch_time_factors": [ 50, 4 ],
+    "book_learn": [ [ "alchemy_basic", 4 ] ],
+    "qualities": [ { "id": "CHEM", "level": 2 }, { "id": "CONCENTRATE", "level": 1 }, { "id": "MANA_FOCUS", "level": 1 } ],
+    "tools": [ [ [ "surface_heat", 10, "LIST" ] ], [ [ "rune_alchemist", -1 ], [ "rune_magus", -1 ] ] ],
+    "components": [
+      [ [ "potion_starter", 5 ] ],
+      [ [ "hairball", 2 ] ],
+      [
+        [ "catfood", 1 ],
+        [ "fish", 1 ],
+        [ "salted_fish", 1 ],
+        [ "fish_cooked", 1 ],
+        [ "fish_smoked", 1 ],
+        [ "dry_fish", 1 ],
+        [ "rehydrated_fish", 1 ],
+        [ "fish_canned", 1 ],
+        [ "fish_pickled", 1 ],
+        [ "fish_fried", 1 ],
+        [ "lutefisk", 1 ]
+      ],
+      [ [ "adderall", 10 ], [ "meth", 6 ], [ "prozac", 15 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "result": "ogres_strength_potion",
+    "category": "CC_ENCHANTED",
+    "subcategory": "CSC_ENCHANTED_POTIONS",
+    "skill_used": "cooking",
+    "skills_required": [ "spellcraft", 5 ],
+    "difficulty": 5,
+    "time": "60 m",
+    "batch_time_factors": [ 50, 4 ],
+    "book_learn": [ [ "alchemy_basic", 4 ] ],
+    "qualities": [ { "id": "CHEM", "level": 2 }, { "id": "CONCENTRATE", "level": 1 }, { "id": "MANA_FOCUS", "level": 1 } ],
+    "tools": [ [ [ "surface_heat", 10, "LIST" ] ], [ [ "rune_alchemist", -1 ], [ "rune_magus", -1 ] ] ],
+    "components": [ [ [ "potion_starter", 5 ] ], [ [ "sinew", 40 ] ], [ [ "adrenal_gland_large", 2 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "eagles_sight_potion",
+    "category": "CC_ENCHANTED",
+    "subcategory": "CSC_ENCHANTED_POTIONS",
+    "skill_used": "cooking",
+    "skills_required": [ "spellcraft", 5 ],
+    "difficulty": 5,
+    "time": "60 m",
+    "batch_time_factors": [ 50, 4 ],
+    "book_learn": [ [ "alchemy_basic", 4 ] ],
+    "qualities": [ { "id": "CHEM", "level": 2 }, { "id": "CONCENTRATE", "level": 1 }, { "id": "MANA_FOCUS", "level": 1 } ],
+    "tools": [ [ [ "surface_heat", 10, "LIST" ] ], [ [ "rune_alchemist", -1 ], [ "rune_magus", -1 ] ] ],
+    "components": [
+      [ [ "potion_starter", 5 ] ],
+      [
+        [ "egg_bird", 5 ],
```

Comment:
iirc there should be a requirement for eggs. i think you should use that.

Reply:
Eggs_bird doesn't respect how large each egg actually is, when the larger eggs are 83ml and the smaller ones are 5ml.